airflow/dags/monitor/monitor.py

```diff
@@ -71,7 +118,8 @@ def check_ui_status():
     endpoint = "https://ask.astronomer.io"
     response = requests.get(endpoint)
     if response.status_code != 200:
-        raise response
+        raise Exception(f"UI check failed with status code: {response.status_code}")
```

Comment:
```suggestion
       response.raise_for_status()
```
what about? this will have better info about error

Reply:
https://github.com/astronomer/ask-astro/pull/98/commits/c62341473efd7ad4ffb18de8255d23e73bd154eb